src/eda/preprocessing/preprocessing.py

```diff
@@ -0,0 +1,52 @@
+import pandas as pd
+import logging as log
+from string import punctuation
+import nltk
+from nltk.stem import PorterStemmer
+from nltk.stem import WordNetLemmatizer
+from nltk.corpus import stopwords
+import re
+
+def drop_nan(data) -> pd.DataFrame:
```

Comment:
```suggestion
def drop_nan(data: pd.DataFrame) -> pd.DataFrame:
```
Same for all other argument